public/Sjofn/practice/sjofn4.html

```diff
@@ -25,5 +25,16 @@
         console.log(Math.sqrt(p));
     }
 
+    let goods = prompt("Enter the name of the object you want to own here:");
```

Comment:
Please, move this line below your function.
**Reason:** functions should be defined near each other and shouldn't have any other instructions between them.